.github/workflows/github-event-notification.yaml

```
@@ -0,0 +1,178 @@
+  pull_request_review_comment:
+    types: [created, edited]
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    name: Send Notifications
+
+    steps:
+      - name: Determine Event Type
+        id: event_info
+        run: |
```

actionlint (1.7.9)

"github.event.issue.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)